packages/core/src/html-entrypoint.ts

```diff
@@ -66,7 +66,7 @@ export class HTMLEntrypoint {
   }
 
   private handledStyles() {
-    let styleTags = [...this.dom.window.document.querySelectorAll('link[rel="stylesheet"]')] as HTMLLinkElement[];
+    let styleTags = [...this.dom.window.document.querySelectorAll('link[rel*="stylesheet"]')] as HTMLLinkElement[];
```

Comment:
This fixes a bug where previously `rel="prefetch stylesheet"` (or similar) in the compat HTML did not get picked up.